cli/src/main/scala/com/lightbend/rp/reactivecli/Main.scala

```diff
@@ -17,8 +17,8 @@
 package com.lightbend.rp.reactivecli
 
 import com.lightbend.rp.reactivecli.argparse.kubernetes.KubernetesArgs
-import com.lightbend.rp.reactivecli.argparse.{ GenerateDeploymentArgs, InputArgs }
-import com.lightbend.rp.reactivecli.docker.{ Config, DockerRegistry }
+import com.lightbend.rp.reactivecli.argparse.{GenerateDeploymentArgs, InputArgs, VersionArgs}
+import com.lightbend.rp.reactivecli.docker.{Config, DockerRegistry}
```

Comment:
accidental format change?

Reply:
Nice catch - I opened #26 to investigate.